install.sh

```diff
@@ -3649,14 +3638,8 @@ find_git_hook_templates() {
             read -r MARK_AS_TEMPLATES </dev/tty
 
             if [ "$MARK_AS_TEMPLATES" = "y" ] || [ "$MARK_AS_TEMPLATES" = "Y" ]; then
-                TEMPLATE_DIR=$(dirname "$TARGET_TEMPLATE_DIR")
```

Comment:
Thats clearly a bug, $TARGE_TEMPLATE_DIR contains `<dir>/hooks` and here we drop the `hooks` part.